django_x509/models.py

```diff
@@ -13,5 +15,19 @@ class Cert(AbstractCert):
     """
     Concrete Cert model
     """
+
+    class Meta(AbstractCert.Meta):
+        abstract = False
+
+
+class CustomCert(AbstractCert):
```

Comment:
If you add this model here it will be added in all the applications which use this library, which is something we don't want, do you agree?

You should add a new django app in the test dir:

```
/tests/customcert/models.py  (here you move CustomCert)
/tests/customcert/tests.py  (here you move TestCustomCert)
```

Then you will need to add the new test app in `settings.INSTALLED_APPS` and it should work.